src/System.Globalization/tests/DateTimeFormatInfo/DateTimeFormatInfoAbbreviatedDayNames.cs

```diff
@@ -23,11 +23,11 @@ public void AbbreviatedDayNames_Set()
             Assert.Equal(newAbbreviatedDayNames, format.AbbreviatedDayNames);
         }
 
-        [Fact]
+        [Fact ActiveIssue(13247, TestPlatforms.AnyUnix)]
         public void AbbreviatedDayNames_Set_Invalid()
         {
             Assert.Throws<ArgumentNullException>("value", () => new DateTimeFormatInfo().AbbreviatedDayNames = null); // Value is null
-            Assert.Throws<ArgumentNullException>("value", () => new DateTimeFormatInfo().AbbreviatedDayNames = new string[] { "1", "2", "3", null, "5", "6", "7" }); // Value has null
+            Assert.Throws<ArgumentNullException>("values[3]", () => new DateTimeFormatInfo().AbbreviatedDayNames = new string[] { "1", "2", "3", null, "5", "6", "7" }); // Value has null
```

Comment:
thanks @jkotas could you please close the issue [https://github.com/dotnet/corefx/issues/13247](https://github.com/dotnet/corefx/issues/13247) then?

---
In reply to: [86171814](https://github.com/dotnet/corefx/pull/13229#discussion_r86171814) [](ancestors = 86171814)